### DISCUSSION TITLE
macOS Status menu

### DISCUSSION THREAD

**Discussion by chris1111** [rejetto/hfs#516] [Show and tell]
I made a simple macOS Status menu
HFS Server.app ➥ [HFS Server.app.zip](https://github.com/rejetto/hfs/files/14548249/HFS.Server.zip)

First edit the script by right clic Show Package Contents /Contents/Resources/script
![Screenshot 1](https://github.com/rejetto/hfs/assets/6248794/42900729-318b-42be-a924-e8fe2a7086e4)

Simply move the app to your Applications folder then make the App at Login Items
![Screenshot 2](https://github.com/rejetto/hfs/assets/6248794/7f9d1c9b-a7aa-4f9b-b807-0cadc6d3988b)

Open the App in the Menu Bar
![Screenshot 3](https://github.com/rejetto/hfs/assets/6248794/7b52149d-a8ca-4667-805f-f30cf54a84ed)

![Screenshot 4](https://github.com/rejetto/hfs/assets/6248794/730b58cb-a5ba-4ece-afa2-c6ee1abe0968)

**Comment by rejetto** [comment]
nice!

**Reply by chris1111** [reply]
Thanks

**Comment by rejetto** [comment]
you should consider distributing it as a plugin: it will have more visibility, and you will be able to know the server directory without user intervention

**Comment by chris1111** [comment]
> you should consider distributing it as a plugin: it will have more visibility, and you will be able to know the server directory without user intervention


How to do? I am very limited regarding your server I run it at 10% everything or more but I would like to be able to do it 😎

**Comment by rejetto** [comment]
well, i'm not sure what it takes in your case, as i didn't have the time to study it, but you should refer to  https://github.com/rejetto/hfs/blob/main/dev-plugins.md

there's a section for publishing as well.
It is possible that your needs could be beyond what the current system is offering. If you identify new needs I can consider expanding it.
